CMakeLists.txt

```diff
@@ -216,9 +225,20 @@ endif()
 if (UNIX)
   add_subdirectory(docs/refman)
   add_subdirectory(pkgconfig)
-  add_subdirectory(samples)
 endif()
 
+add_subdirectory(samples)
+
+if (WIN32)
+  install(FILES ./scripts/clangw ./scripts/llvm-arw
```

Comment:
We set `OE_SCRIPTSDIR` above but aren't using it here (which, I mean, I wouldn't anyway, but what is `OE_SCRIPTSDIR` for?)